filebeat/docs/multiline.asciidoc

```diff
@@ -46,10 +46,9 @@ To consolidate these lines into a single event in Filebeat, use the following mu
 
 [source,json]
 -------------------------------------------------------------------------------------
-multiline:
-  pattern: '^[[:space:]]'
-  negate: false
-  match: after
+multiline.pattern: '^[[:space:]]'
```

Comment:
I agree. The good news is that this only happens for the filebeat prospectors. For everything else, after the dot notation change, the config file sample stand for themselves.